need to use collections.abc.Sequence instead of collections.Sequence in python 3.10 and later

lucasw opened the pull request.

lianghongzhuo commented:
could you also make your pr at https://github.com/ros-o/ros_numpy?

lucasw commented:
It's already in obese-devel on the ros-o fork @lianghongzhuo (maybe only as of this morning) - I'll leave this open here in case it comes back under maintenance but anyone else with an issue or pr should be pointed at ros-o


Back linking to https://github.com/ros-o/ros-o/discussions/7#discussioncomment-5227807

lianghongzhuo commented:
Thank you!

eric-wieser reviewed (commented):
Strictly this breaks Python 2; I think probably support for python2 should be removed from the package.xml at the same time as this change.

lucasw commented:
> Strictly this breaks Python 2; I think probably support for python2 should be removed from the package.xml at the same time as this change.

Rebased and removed the python 2 from package.xml
